systers_portal/community/migrations/0001_initial.py

```diff
@@ -24,7 +24,7 @@ class Migration(migrations.Migration):
                 ('facebook', models.URLField(max_length=255, verbose_name=b'Facebook', blank=True)),
                 ('googleplus', models.URLField(max_length=255, verbose_name=b'Google+', blank=True)),
                 ('twitter', models.URLField(max_length=255, verbose_name=b'Twitter', blank=True)),
-                ('community_admin', models.ForeignKey(related_name=b'community', verbose_name=b'Community admin', to='users.SystersUser')),
```

Comment:
It took some time to find this bug.

@todipratik you should never edit manually generated migrations. See [the documentation on that](http://systers-portal.readthedocs.org/en/develop/develop/architecture.html).

Remove changes in this file and run `makemigrations`, which will create a new file in the migrations folder with the rename. Commit this file. It helps others to migrate their forks as well.